README.md

```diff
@@ -218,8 +210,7 @@ I would like to thank Microverse for providing me with the reading material that
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Great job working on requested changes, but as per the screenshot below, I am still getting error 404 when I click on the MIT link. Kindly fix this issue.
![image](https://user-images.githubusercontent.com/62849872/218962781-7e96f409-e27d-4738-987b-122de40707b6.png)